src/routes/votings/[id]/+page.svelte

```diff
@@ -1,9 +1,495 @@
+	<Breadcrumb
+		noTrailingSlash
+		class="w-full px-4 py-2 body-compact-01 [&>.bx--breadcrumb]:flex text-ellipsis overflow-hidden truncate"
+	>
+		<BreadcrumbItem href="#">หน้าแรก</BreadcrumbItem>
+		<BreadcrumbItem href="#">การลงมติ</BreadcrumbItem>
+		<BreadcrumbItem>{data.voting.title}</BreadcrumbItem>
+	</Breadcrumb>
+	<div
+		class="flex flex-col gap-y-4 md:gap-y-8 w-full {getBillStatusColor(
+			data.voting.result
+		)} px-4 md:px-12 py-8 md:py-16"
+	>
+		<div class="flex flex-col">
+			<h1 class="fluid-heading-05">{data.voting.title}</h1>
+			<div class="flex items-center text-gray-60 gap-x-1">
+				<p class="flex-none heading-01">ชื่อทางการ</p>
+				<p class="flex-initial body-01 truncate">{data.relatedBill.nickname}</p>
+			</div>
+			<div class="flex items-center text-01 gap-x-1">
+				<VotingResultTag result={data.voting.result} isLarge />
+				<p class="heading-compact-02">
+					เห็นด้วย {data.results.find((v) => v.voteOption === 'เห็นด้วย')?.total || 0}/<span
+						class="text-gray-60"
+						>{data.results.find((v) => v.voteOption === 'ไม่เห็นด้วย')?.total || 0}</span
+					>
+				</p>
+			</div>
+		</div>
+		<div class="flex flex-col gap-y-4 md:gap-y-0 md:flex-row md:gap-x-16">
+			<div class="flex flex-col flex-1">
+				<div class="flex justify-between">
+					<div>
+						<p class="heading-01">วันที่</p>
+						<p class="body-01">{dateConvertor(data.voting.date)}</p>
+					</div>
+					<div>
+						<p class="heading-01">ประเภทการประชุม</p>
+						<p class="body-01">{data.voting.meetingType}</p>
+					</div>
+					<div>
+						<p class="heading-01">องค์ประชุม</p>
+						<p class="body-01">
+							{#each data.voting.participatedAssembleIds as id}
+								<p class="underline">{id}</p>
+							{/each}
+						</p>
+					</div>
+				</div>
+				<div class="w-full my-4 h-[1px] bg-gray-20" />
+				<p class="heading-01">สรุปเนื้อหา</p>
+				<p class="body-01">{data.relatedBill.description}</p>
+				{#if data.relatedBill}
+					<div class="flex items-center gap-x-1 mt-4">
+						<p class="heading-01">หมวด</p>
+						{#each data.voting.categories as category}
+							<BillCategoryTag isLarge label={category} />
+						{/each}
+					</div>
+					<p class="heading-01 mt-4 mb-1">ดูเส้นทางของร่างกฏหมายนี้</p>
+					<BillCard
+						class={isMobile ? 'w-full' : 'w-auto'}
+						orientation={isMobile ? 'portrait' : 'landscape'}
+						title={data.relatedBill.title}
+						nickname={data.relatedBill.nickname}
+						proposedOn={data.relatedBill.proposedOn}
+						status={data.relatedBill.status}
+						currentState={data.relatedBill.status}
+						daySinceProposed={getDiffDays(data.relatedBill.proposedOn)}
+						billUrl={data.relatedBill.title}
+						proposedBy={{
+							avatar: data.relatedBill.proposedLedByPolitician?.avatar || '',
+							name: data.relatedBill.proposedLedByPolitician?.id.replace('-', ' ') || '',
+							description:
+								data.relatedBill.proposedLedByPolitician?.assemblyRoles[0].assembly.id || ''
+						}}
```

Comment:
1. Better join `firstname` and `lastname`. The `id` doesn't need to always derive from the name.
2. According to Figma, we have 3 variations of proposer: `proposedLedByPolitician`, `proposedByAssembly`, and `proposedByPeople`. It can be one of these 3 so you need to handle another 2 cases too.